<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="7" ignored="0" total="11" passed="4">
  <reporter-output>
  </reporter-output>
  <suite name="Surefire suite" duration-ms="113530" started-at="2020-10-24T19:57:12Z" finished-at="2020-10-24T19:59:06Z">
    <groups>
    </groups>
    <test name="Surefire test" duration-ms="113530" started-at="2020-10-24T19:57:12Z" finished-at="2020-10-24T19:59:06Z">
      <class name="com.carbonite.qa.tests.SignUpPageTest">
        <test-method status="PASS" signature="loggersetUp()[pri:0, instance:com.carbonite.qa.tests.SignUpPageTest@2669b199]" name="loggersetUp" is-config="true" duration-ms="5" started-at="2020-10-24T19:58:07Z" finished-at="2020-10-24T19:58:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loggersetUp -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.carbonite.qa.tests.SignUpPageTest@2669b199]" name="setUp" is-config="true" duration-ms="7509" started-at="2020-10-24T19:58:07Z" finished-at="2020-10-24T19:58:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="fillSignUpFormWithValidTestData(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:com.carbonite.qa.tests.SignUpPageTest@2669b199]" name="fillSignUpFormWithValidTestData" duration-ms="1808" started-at="2020-10-24T19:58:15Z" data-provider="PositiveSignUpDataFromExcelFile" finished-at="2020-10-24T19:58:17Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[manvir..singh9@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[manvir..singh9@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Manager@123]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Manager@123]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Download] but found [Try Carbonite Computer Backup Free For 15 Days | Carbonite]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Download] but found [Try Carbonite Computer Backup Free For 15 Days | Carbonite]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.carbonite.qa.tests.SignUpPageTest.fillSignUpFormWithValidTestData(SignUpPageTest.java:64)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fillSignUpFormWithValidTestData -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.carbonite.qa.tests.SignUpPageTest@2669b199]" name="tearDown" is-config="true" duration-ms="752" started-at="2020-10-24T19:58:17Z" finished-at="2020-10-24T19:58:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.carbonite.qa.tests.SignUpPageTest@2669b199]" name="setUp" is-config="true" duration-ms="7599" started-at="2020-10-24T19:58:18Z" finished-at="2020-10-24T19:58:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="fillSignUpFormWithValidTestData(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:com.carbonite.qa.tests.SignUpPageTest@2669b199]" name="fillSignUpFormWithValidTestData" duration-ms="1723" started-at="2020-10-24T19:58:25Z" data-provider="PositiveSignUpDataFromExcelFile" finished-at="2020-10-24T19:58:27Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[manvir..singh91@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[manvir..singh91@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Manager@123]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Manager@123]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Download] but found [Try Carbonite Computer Backup Free For 15 Days | Carbonite]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Download] but found [Try Carbonite Computer Backup Free For 15 Days | Carbonite]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.carbonite.qa.tests.SignUpPageTest.fillSignUpFormWithValidTestData(SignUpPageTest.java:64)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fillSignUpFormWithValidTestData -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.carbonite.qa.tests.SignUpPageTest@2669b199]" name="tearDown" is-config="true" duration-ms="745" started-at="2020-10-24T19:58:27Z" finished-at="2020-10-24T19:58:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.carbonite.qa.tests.SignUpPageTest@2669b199]" name="setUp" is-config="true" duration-ms="7525" started-at="2020-10-24T19:58:28Z" finished-at="2020-10-24T19:58:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="fillSignUpFormWithValidTestData(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:com.carbonite.qa.tests.SignUpPageTest@2669b199]" name="fillSignUpFormWithValidTestData" duration-ms="1545" started-at="2020-10-24T19:58:35Z" data-provider="PositiveSignUpDataFromExcelFile" finished-at="2020-10-24T19:58:37Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[manvir..singh92@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[manvir..singh92@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Manager@123]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Manager@123]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Download] but found [Try Carbonite Computer Backup Free For 15 Days | Carbonite]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Download] but found [Try Carbonite Computer Backup Free For 15 Days | Carbonite]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.carbonite.qa.tests.SignUpPageTest.fillSignUpFormWithValidTestData(SignUpPageTest.java:64)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fillSignUpFormWithValidTestData -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.carbonite.qa.tests.SignUpPageTest@2669b199]" name="tearDown" is-config="true" duration-ms="778" started-at="2020-10-24T19:58:37Z" finished-at="2020-10-24T19:58:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.carbonite.qa.tests.SignUpPageTest@2669b199]" name="setUp" is-config="true" duration-ms="7397" started-at="2020-10-24T19:58:38Z" finished-at="2020-10-24T19:58:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="fillSignUpFormWithNegativeTestData(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:2, instance:com.carbonite.qa.tests.SignUpPageTest@2669b199]" name="fillSignUpFormWithNegativeTestData" duration-ms="1063" started-at="2020-10-24T19:58:45Z" data-provider="NegativeSignUpDataFromExcelFile" finished-at="2020-10-24T19:58:46Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[manvir456@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[manvir123@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[password@1234]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[password@1234]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fillSignUpFormWithNegativeTestData -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.carbonite.qa.tests.SignUpPageTest@2669b199]" name="tearDown" is-config="true" duration-ms="1540" started-at="2020-10-24T19:58:46Z" finished-at="2020-10-24T19:58:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.carbonite.qa.tests.SignUpPageTest@2669b199]" name="setUp" is-config="true" duration-ms="8168" started-at="2020-10-24T19:58:48Z" finished-at="2020-10-24T19:58:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="fillSignUpFormWithNegativeTestData(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:2, instance:com.carbonite.qa.tests.SignUpPageTest@2669b199]" name="fillSignUpFormWithNegativeTestData" duration-ms="767" started-at="2020-10-24T19:58:56Z" data-provider="NegativeSignUpDataFromExcelFile" finished-at="2020-10-24T19:58:57Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[manvir456@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[manvir456@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[password@123]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[password@1234]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fillSignUpFormWithNegativeTestData -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.carbonite.qa.tests.SignUpPageTest@2669b199]" name="tearDown" is-config="true" duration-ms="770" started-at="2020-10-24T19:58:57Z" finished-at="2020-10-24T19:58:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.carbonite.qa.tests.SignUpPageTest@2669b199]" name="setUp" is-config="true" duration-ms="7773" started-at="2020-10-24T19:58:57Z" finished-at="2020-10-24T19:59:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="test()[pri:3, instance:com.carbonite.qa.tests.SignUpPageTest@2669b199]" name="test" duration-ms="1" started-at="2020-10-24T19:59:05Z" finished-at="2020-10-24T19:59:05Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:568)
at org.testng.Assert.assertEquals(Assert.java:578)
at com.carbonite.qa.tests.SignUpPageTest.test(SignUpPageTest.java:100)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.carbonite.qa.tests.SignUpPageTest@2669b199]" name="tearDown" is-config="true" duration-ms="728" started-at="2020-10-24T19:59:05Z" finished-at="2020-10-24T19:59:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.carbonite.qa.tests.SignUpPageTest -->
      <class name="com.carbonite.qa.tests.CompareBackUpPlanTest">
        <test-method status="PASS" signature="loggersetUp()[pri:0, instance:com.carbonite.qa.tests.CompareBackUpPlanTest@6574b225]" name="loggersetUp" is-config="true" duration-ms="4" started-at="2020-10-24T19:57:56Z" finished-at="2020-10-24T19:57:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loggersetUp -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.carbonite.qa.tests.CompareBackUpPlanTest@6574b225]" name="setUp" is-config="true" duration-ms="9121" started-at="2020-10-24T19:57:56Z" finished-at="2020-10-24T19:58:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="getTitleOfThePage()[pri:1, instance:com.carbonite.qa.tests.CompareBackUpPlanTest@6574b225]" name="getTitleOfThePage" duration-ms="282" started-at="2020-10-24T19:58:06Z" finished-at="2020-10-24T19:58:06Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Personal Cloud Backup Pricing, Plans & Features | Carbonite] but found [Backup Product Select | Carbonite]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Personal Cloud Backup Pricing, Plans & Features | Carbonite] but found [Backup Product Select | Carbonite]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.carbonite.qa.tests.CompareBackUpPlanTest.getTitleOfThePage(CompareBackUpPlanTest.java:36)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getTitleOfThePage -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.carbonite.qa.tests.CompareBackUpPlanTest@6574b225]" name="tearDown" is-config="true" duration-ms="934" started-at="2020-10-24T19:58:06Z" finished-at="2020-10-24T19:58:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.carbonite.qa.tests.CompareBackUpPlanTest -->
      <class name="com.carbonite.qa.tests.AllProductsPageTest">
        <test-method status="PASS" signature="loggersetUp()[pri:0, instance:com.carbonite.qa.tests.AllProductsPageTest@32e6e9c3]" name="loggersetUp" is-config="true" duration-ms="64" started-at="2020-10-24T19:57:12Z" finished-at="2020-10-24T19:57:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loggersetUp -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.carbonite.qa.tests.AllProductsPageTest@32e6e9c3]" name="setUp" is-config="true" duration-ms="10636" started-at="2020-10-24T19:57:12Z" finished-at="2020-10-24T19:57:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="test1()[pri:0, instance:com.carbonite.qa.tests.AllProductsPageTest@32e6e9c3]" name="test1" duration-ms="6" started-at="2020-10-24T19:57:23Z" finished-at="2020-10-24T19:57:23Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:568)
at org.testng.Assert.assertEquals(Assert.java:578)
at com.carbonite.qa.tests.AllProductsPageTest.test1(AllProductsPageTest.java:33)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.carbonite.qa.tests.AllProductsPageTest@32e6e9c3]" name="tearDown" is-config="true" duration-ms="705" started-at="2020-10-24T19:57:23Z" finished-at="2020-10-24T19:57:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.carbonite.qa.tests.AllProductsPageTest@32e6e9c3]" name="setUp" is-config="true" duration-ms="9210" started-at="2020-10-24T19:57:24Z" finished-at="2020-10-24T19:57:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="learnMoreClickTest()[pri:1, instance:com.carbonite.qa.tests.AllProductsPageTest@32e6e9c3]" name="learnMoreClickTest" duration-ms="1273" started-at="2020-10-24T19:57:33Z" finished-at="2020-10-24T19:57:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- learnMoreClickTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.carbonite.qa.tests.AllProductsPageTest@32e6e9c3]" name="tearDown" is-config="true" duration-ms="909" started-at="2020-10-24T19:57:34Z" finished-at="2020-10-24T19:57:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.carbonite.qa.tests.AllProductsPageTest@32e6e9c3]" name="setUp" is-config="true" duration-ms="9153" started-at="2020-10-24T19:57:35Z" finished-at="2020-10-24T19:57:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="learnMoreClickTest()[pri:1, instance:com.carbonite.qa.tests.AllProductsPageTest@32e6e9c3]" name="learnMoreClickTest" duration-ms="1220" started-at="2020-10-24T19:57:44Z" finished-at="2020-10-24T19:57:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- learnMoreClickTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.carbonite.qa.tests.AllProductsPageTest@32e6e9c3]" name="tearDown" is-config="true" duration-ms="849" started-at="2020-10-24T19:57:45Z" finished-at="2020-10-24T19:57:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.carbonite.qa.tests.AllProductsPageTest -->
      <class name="com.carbonite.qa.tests.BuisnessSolutionPageTest">
        <test-method status="PASS" signature="loggersetUp()[pri:0, instance:com.carbonite.qa.tests.BuisnessSolutionPageTest@5056dfcb]" name="loggersetUp" is-config="true" duration-ms="5" started-at="2020-10-24T19:57:46Z" finished-at="2020-10-24T19:57:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loggersetUp -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.carbonite.qa.tests.BuisnessSolutionPageTest@5056dfcb]" name="setUp" is-config="true" duration-ms="9238" started-at="2020-10-24T19:57:46Z" finished-at="2020-10-24T19:57:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="getTitleOfThePage()[pri:1, instance:com.carbonite.qa.tests.BuisnessSolutionPageTest@5056dfcb]" name="getTitleOfThePage" duration-ms="7" started-at="2020-10-24T19:57:56Z" finished-at="2020-10-24T19:57:56Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Small business cloud backup for your business data | Carbonite] but found []]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Small business cloud backup for your business data | Carbonite] but found []
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.carbonite.qa.tests.BuisnessSolutionPageTest.getTitleOfThePage(BuisnessSolutionPageTest.java:34)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getTitleOfThePage -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.carbonite.qa.tests.BuisnessSolutionPageTest@5056dfcb]" name="tearDown" is-config="true" duration-ms="854" started-at="2020-10-24T19:57:56Z" finished-at="2020-10-24T19:57:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.carbonite.qa.tests.BuisnessSolutionPageTest -->
    </test> <!-- Surefire test -->
  </suite> <!-- Surefire suite -->
</testng-results>
